.github/ISSUE_TEMPLATE/BUG-ISSUE.yml

```
@@ -0,0 +1,49 @@
+name: Bug Report
+description: Create a report to help us improve
+labels: ["Type: Bug"]
+body:
+  - type: dropdown
+    id: version
+    attributes:
+      label: Version
+      description: What version of Gloo Mesh are you using?
+      options:
+        - 1.3.x 
+        - 1.2.x
```

What do you think about annotating these to make it clear to users, just incase they assume the top one is the latest one they installed from default via meshctl?
```suggestion
        - 1.3.x (beta)
        - 1.2.x (latest stable)
```

---

I like that idea. We should probably come up with a README or something that talks about what needs to change when a release is cut (adding new version, adjusting annotations, etc.). Ideal world, that would be automated as part of a release. :-)